packages/app-builder/src/routes/_builder+/scenarios+/$scenarioId+/i+/$iterationId+/_edit-view+/trigger.tsx

```diff
@@ -82,7 +82,8 @@ export async function loader({ request, params }: LoaderFunctionArgs) {
   ]);
 
   return json({
-    identifiers,
+    databaseAccessors: accessors.databaseAccessors,
+    payloadAccessors: accessors.payloadAccessors,
```

Comment:
flatten the strucutre, it do not make sense from the frontend perspective to have nesting here (this structure was due to the first implementation, were we only had simple nodes)